Sources_v3/StreamChatUI/ChatChannel/ChatMessageBubbleView.swift

```diff
@@ -105,15 +106,15 @@ open class ChatMessageBubbleView<ExtraData: ExtraDataTypes>: View, UIConfigProvi
             ]
         }
 
-        layoutConstraints[[.text, .images]] = [
-            imageGallery.leadingAnchor.constraint(equalTo: leadingAnchor),
-            imageGallery.trailingAnchor.constraint(equalTo: trailingAnchor),
-            imageGallery.topAnchor.constraint(equalTo: topAnchor),
-            imageGallery.widthAnchor.constraint(equalToConstant: UIScreen.main.bounds.width * 0.6),
+        layoutConstraints[[.text, .attachments]] = [
```

Comment:
That wrong for `Link preview` which is attachment as well, but should be after text.

Or you plan to not handle links as attachments?

Reply:
Yep, intended to have a separate UI component for previews